src/InvestmentManager.sol

```diff
@@ -26,10 +26,10 @@ interface VaultLike is IERC20 {
     function trancheId() external view returns (bytes16);
     function asset() external view returns (address);
     function share() external view returns (address);
-    function emitDepositClaimable(address owner, uint256 assets, uint256 shares) external;
-    function emitRedeemClaimable(address owner, uint256 assets, uint256 shares) external;
-    function emitCancelDepositClaimable(address owner, uint256 assets) external;
-    function emitCancelRedeemClaimable(address owner, uint256 shares) external;
+    function onDepositClaimable(address owner, uint256 assets, uint256 shares) external;
```

Comment:
A bit nicer imo since in theory this is a generic hook, that the vault can decide to implement and do whatever they want with, which now is just emitting an event